.husky/pre-commit

```diff
@@ -1 +1,6 @@
-lint-staged
+npm run lint
+npm run prettier:check
+npm run build
+npm run test-check-coverage
+npm run test-headless
```

Comment:
Could these be moved to the `pre-push` hook? This seems a bit heavy/slow for every commit.

Reply:
I understand, and see what you mean in terms of developer pain point.  I reverted the change for husky/lint-stage to the way it used to be.  The Github workflow already runs the test-check-coverage script as well, so adding it to the client side just slows things down.